services/slam/builtin/builtin_test.go

```diff
@@ -475,9 +481,21 @@ func TestGeneralNew(t *testing.T) {
 		logger := golog.NewTestLogger(t)
 		attrCfg := &builtin.AttrConfig{}
 		_, err := createSLAMService(t, attrCfg, logger, false, false)
-		test.That(t, err, test.ShouldBeError,
-			errors.Errorf("runtime slam config error: "+
-				"%v algorithm specified not in implemented list", attrCfg.Algorithm))
```

Comment:
This is now caught by the `Validate` function, testing for this error still exists in `slam_ext_test.go`